docs(js): Add gatsby source maps instructions (#3389)
getsentry/sentry-docs: src/platforms/javascript/common/sourcemaps/index.mdx

};
```

</PlatformSection>

<PlatformSection supported={["javascript.gatsby"]}>

```javascript {filename:webpack.config.js}
const SentryWebpackPlugin = require("@sentry/webpack-plugin");

exports.onCreateWebpackConfig = ({ stage, actions }) => {
  if (stage === "build-javascript") {
    actions.setWebpackConfig({
      plugins: [
        new SentryWebpackPlugin({
          // sentry-cli configuration
          authToken: process.env.SENTRY_AUTH_TOKEN,
          org: "___ORG_SLUG___",
          project: "___PROJECT_SLUG___",

          // webpack specific configuration
          include: "./public",
          release: process.env.SENTRY_RELEASE,
        }),
      ],
    });
  }
};
```

> Instead of using `SENTRY_RELEASE` directly, you can change it for an [appropriate env variable](https://github.com/getsentry/sentry-javascript/blob/master/packages/gatsby/gatsby-node.js) based on the environment you are going to deploy your application on.

Reviewer: ```suggestion
You can change `SENTRY_RELEASE` directly to an [appropriate env variable](https://github.com/getsentry/sentry-javascript/blob/master/packages/gatsby/gatsby-node.js) based on the environment on which you are going to deploy your application.
```

</PlatformSection>

<PlatformSection supported={["javascript.vue"]}>

In Vue 2.x, you should use `vue.config.js` instead of `webpack.config.js`, and use `include: "./dist"` instead of `include: "."`

</PlatformSection>

> Always make sure that the release value you are providing during Source Maps upload, matches the value used in the `Sentry.init` call.

Reviewer: ```suggestion
Note: The release value provided during source maps upload must match the value in the `Sentry.init` call.
```
